src/components/Label.js

```diff
@@ -18,14 +18,19 @@ const Label = props => {
     align,
     xs,
     className,
+    color,
     ...otherProps
   } = props;
 
   // check if column width has been specified, use alignment attribute if so
   const cols = colWidths.filter(colWidth => props[colWidth] || props[colWidth] === '');
 
   const alignClass = align && alignMap[align];
-  const classes = classNames(className, cols.length && alignClass);
+  const classes = classNames(
+    className,
+    cols.length && alignClass,
+    color && `text-${color}`
```

Comment:
Is it more common to put `className` last? It seems likely that, if the user specifies `className` explicitly, that should just override any other option.

```suggestion
    const classes = classNames(
      cols.length && alignClass,
      color && `text-${color}`,
      className
    )
```

Reply:
Actually, ignore this -- it's irrelevant since classes are always additive.